sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+
+Once all of the outstanding UTXOs have been consumed by BTC-transfers, the new
+Stackers begin handling deposits and withdraws for the next reward cycle.
+
+### Recovery of BTC Fees
+
+The transfer is complete once all of the old wallet's UTXOs (both unused and
+pending) are consumed by BTC-transfer transactions.  However,
+the total BTC in the sBTC wallet will be slightly less than the
+total amount of sBTC.  This is because (1) the old Stackers will have had to pay
+transaction fees to send the BTC-transfer transactions, and (2) Stackers
+may have had to RBF a stuck BTC redeem transaction.
+
+These losses due to BTC fees are distributed across sBTC accounts lazily when the user next
+deposits BTC, transfers the sBTC, or attempts to redeem it.  The `.sbtc` contract keeps track
+of the total supply of sBTC, the balance of each account, and the number of
+missing satoshis, so it has enough information to make this happen.  When the user attempts
+to do any of the above with sBTC, the `.sbtc` contract will burn an amount of sBTC equal to
+the following:
+
+```
+min(1, (sbtc_balance * missing_satoshis) / sbtc_supply)
+```
+
+In other words, the sBTC holder will pay a pro-rata share of the missing BTC
+fees, or it will pay a single satoshi of it.  An sBTC holder pays at most once
+per reward cycle.  Eventually, as long as there is
+sBTC activity on-chain, the number of sBTC in circulation will be made equal to
+the number of satoshis in the sBTC wallet.  If this fails to happen by the end of
+the reward cycle, then the sBTC system freezes and a Stacker may need to
+manually pay BTC into the sBTC wallet to recover the fee (see "System Recovery
+from Frozen Wallet").
+
+### Restarting In-Flight Transactions
+
+If a BTC-transfer happens while there are in-flight redemption transactions, then the old
+Stackers must double-spend the in-flight redemption transactions.  The `.sbtc`
+contract identifies which in-flight redemption transactions would be affected by
+observing which UTXOs the BTC-transfer transaction consumes.  Each affected in-flight
+redemption transaction would be marked as cancelled, and the user's sBTC
+unlocks.  The user will need to try the request again.
+
+### Transfer Fees
+
+The peg wallet will need to be transferred every reward cycle, which is an unbounded number of
+transfers with no tie to the numebr deposit and withdrawal requests. This means there is
+no way to charge a fee that, at the moment of deposit or withdrawal, is
+guaranteed to adequately pay for the peg transfer fees for the duration of the pegged
+Bitcoin's tenure.
+
+To pay for peg wallet transfer fees without dipping into user funds, this SIP proposes
+the following mitigation strategies.
+
+1. During the prepare phase, the traditionally burned bitcoin will instead be placed into the peg wallet. See [SIP-001][SIP-001-LINK]
+2. The protocol will maintain a warchest that can receive donations and be siphoned from if the wallet budget for transfer fees runs dry.
```

Comment:
This should be altered to make it clear that the "warchest" is just a concept that maps to "the amount of BTC in the peg wallet that's not accounted for in the sBTC system"

Reply:
How about a term that doesn't have "war" connotations? Like "treasury" or "surplus" or something like that.